packages/ui/src/components/Button/index.tsx

```diff
@@ -4,22 +4,30 @@ import React from 'react';
 import * as styles from './index.module.scss';
 
 export type Props = {
+  htmlType?: 'button' | 'submit' | 'reset';
   isDisabled?: boolean;
   className?: string;
-  value?: string;
+  children: string; // TODO: make it i18nKey with optional params
+  type?: 'primary' | 'secondary';
   onClick?: React.MouseEventHandler;
 };
 
-const Button = ({ isDisabled, className, value, onClick }: Props) => {
-  return (
-    <input
-      disabled={isDisabled}
-      className={classNames(styles.button, className)}
-      type="button"
-      value={value}
-      onClick={onClick}
-    />
-  );
-};
+const Button = ({
+  htmlType = 'button',
+  type = 'primary',
+  isDisabled,
```

Comment:
nit: `disabled` should be good enough

Reply:
react props lint rule recommend giving a boolean liked name.  One possible way is to extend the whole riginal button props.  Keep isDisabled for now?

Reply:
ok